src/components/summary/_macro.njk

```diff
@@ -120,7 +120,7 @@
                     {% endif %}
 
                     {% if group.summaryLink is defined and group.summaryLink %}
-                        <div class="u-mb-l u-pt-s{% if group.placeholderText is not defined and group.rows is defined and group.rows %} u-bt{% endif %}">
+                        <div class="u-pt-s{% if group.placeholderText is not defined and group.rows is defined and group.rows %} u-bt{% endif %}">
```

Comment:
I think this padding and border shouldn't use util classes, and instead should be css, so I would add a class for this link (we may want to style it in future) now that we know it's specific use e.g. `summary__add-link`